src/Couchbase.Lite.Tests.Shared/TLSIdentityTest.cs

```diff
@@ -138,7 +161,7 @@ public void TestCreateIdentityWithNoAttributesOrEmptyAttributes()
             id.Should().BeNull();
 
             // Create id with empty Attributes
-            Action badAction = (() => TLSIdentity.CreateIdentity(true,
+            Action badAction = (() => TLSIdentity.CreateIdentity(KeyUsages.ServerAuth,
```

Comment:
i just hate the way python formats things.
There have been multiple studies that demonstrate that putting important things out in the right gutter causes them to be overlooked.  Grrrr....